src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -143,22 +143,32 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
                 options: {
                     gradeNames: ["fluid.prefs.uiEnhancerRelay", "fluid.prefs.arrowScrolling"],
                     // ensure that model and applier are available to users at top level
-                    model: "{separatedPanel}.model",
+                    model: {
+                        preferences: "{separatedPanel}.model.preferences",
+                        panelIndex: "{separatedPanel}.model.panelIndex",
+                        panelMaxIndex: "{separatedPanel}.model.panelMaxIndex",
+                        local: {
+                            panelIndex: "{that}.model.panelIndex"
```

Comment:
What is the distinction between the local panelIndex and the top-level one?

Reply:
@amb26 this is the setup for what is tracked for saving between the local and remote models, as part of the fluid.remoteModelComponent.

Reply:
OK - it would be useful to add a comment on line 150 reminding the user that this structuring arises through the use of fluid.remoteModelComponent - and that what it is for is arranging for this to be persisted and synchronised there

Reply:
@amb26 that makes sense. I've added a comment about it.